### ISSUE TITLE
[FEATURE]Boss Themes

### ISSUE THREAD

**Issue by NoRestForWicked** [mariumbacchus/Soulslike-Weaponry#166]
My problem is that the epic fights I encounter need an epic boss theme to go along side them 🔥 

I have created a currently unofficial theme for the Returning Knight/Fallen Icon. The main composition is done, but slight changes such as volume and more can still be done.

If you have any suggestions for it or requests I can change it as you wish.
It is heavily based off the Terraria Calamity Mod boss themes, Dark souls, and Elden ring.

If you don't like the drums, it still sounds _really_ souls-like without them.

Let me know what you think, **all** constructive criticism is requested and politely accepted :D
Also, you don't have to accept it, but would you be fine with me using it as the boss theme in my own custom boss music mod?

Video link (This will NOT be the video used. I will make a cool boss theme video like I did for the wither): https://youtu.be/H2l1-x3-dNA

**Comment by mariumbacchus** [comment]
This is sick, I'd love to implement this. I am also a big fan of Calamity and souls games soundtrack, and I'd say this fits very well. I honestly don't know what you could do to make it better besides fixing sound quality and all that you mentioned yourself. Like I said, I would really like to add this when it is finished, just let me know how I can credit you when you are done!

**Comment by NoRestForWicked** [comment]
Absolutely, the track is now done but I'm working on an alternate version that sounds more souls-like and has less intensive drums. If you want, both fallen icon and returning knight can share the theme, or one can get the original one (intense) and the other get the less intense one. 

I will work on the video and send it out, and for crediting, I'm sure just sharing my name as the creator of the music will work.

If you have discord, what's your user? It'll make contact much easier :D

I also know how to implement it if you would like to know it or if you have your own way to do it that's cool too

**Comment by NoRestForWicked** [comment]
Video is OUT: https://youtu.be/oKN62bO9aRQ

I have an idea for the music, I've made a mod (still testing) that implements the boss theme (I can add on to it aswell), and since adding the music to the main mod would increase its size drastically, we should upload this as a seperate mod (SoulsLike Music Mod)

**Comment by mariumbacchus** [comment]
Good idea with the mod, contact me on curseforge for the discord profile so I don't leak it to other people here too.

**Comment by NoRestForWicked** [comment]
done👍

**Comment by NoRestForWicked** [comment]
any update 😄 ?

**Comment by mariumbacchus** [comment]
Will be adding it this next update!

**Comment by mariumbacchus** [comment]
Finally implemented for the next update for Fallen Icon, feel free to re-open if you want to provide for other bosses too.
